tests/main/views/test_buyers.py

```diff
@@ -92,7 +92,7 @@ def setup_method(self, method):
     def test_draft_briefs_section(self, data_api_client, find_briefs_mock):
         data_api_client.find_briefs.return_value = find_briefs_mock
 
-        res = self.client.get("/buyers")
+        res = self.client.get("/buyers/dos-requirements")
         tables = html.fromstring(res.get_data(as_text=True)).xpath('//table')
```

Comment:
@samuelhwilliams / @galund - could I just do the following

```
buyer_dashboard_url = "/buyers"

import flask_featureflags;
if flask_featureflags.is_active('DIRECT_AWARD_PROJECTS'):
    buyer_dashboard_url = "/buyers/dos-requirements"
```

Reply:
Doing it that way is fine by me.